perl_lib/EPrints/Plugin/Screen/EPrint/Move.pm

```diff
@@ -29,7 +29,7 @@ sub new
 { place => "eprint_actions_bar_archive", action => "move_buffer", position => 100, },
 { place => "eprint_actions_bar_archive", action => "move_deletion", position => 100, },
 { place => "eprint_actions_bar_deletion", action => "move_archive", position => 100, },
-{ place => "eprint_review_actions", action => "move_archive", postion => 200, },
+{ place => "eprint_review_actions", action => "move_archive", position => 200, },
```

Comment:
This particular typo fix is probably higher priority than everything else in this PR, since it's actually removing a bug. Even if it's a bug nobody has reported before.